### PR TITLE
Location search function

### PR DESCRIPTION
**Pull request by xiaopow**

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/posts/live_feed.js.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+app.controller('LiveFeedCtrl', ['$scope', '$http', function($scope, $http) {
+  
+  postId = 1
```
Comment (harrychenca) [inline app/assets/javascripts/angular/controllers/posts/live_feed.js.erb]: What is postId used for?

File: app/assets/javascripts/angular/controllers/posts/live_feed.js.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+                            
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000/";
+  
+  $http.get(url + 'live_feed/' + postId).success(function(response) {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/posts/live_feed.js.erb:7]: Are you only retrieving one post for the whole live feed page?

File: config/routes.rb [hunk 3]
```diff
@@ -25,6 +25,8 @@
   
   resources :posts
 
+  get 'live_feed/:id' => 'posts#around'
```
Comment (h8rry) [inline config/routes.rb]: The naming of the routes should be more consistent with the method names.

For example, if the url is live_feed, then you should have a `live_feed` method for that route, but currently it's a helper function.

**Comment by h8rry** [comment]
Looks good :sunrise:

**Pull request merged**